pkgs/development/compilers/yosys/default.nix

```diff
@@ -8,14 +8,14 @@ with builtins;
 
 stdenv.mkDerivation rec {
   pname = "yosys";
-  version = "2019.08.21";
+  version = "2019.08.31";
```

Comment:
Nit: Yosys 0.9 was [released](https://github.com/YosysHQ/yosys/releases/tag/yosys-0.9) 5 days ago. Shouldn't we use that instead?

Reply:
That doesn't include `-abc9`, and the QoS with plain `-abc` is laughably bad, at least on iCE40.

Reply:
Ah! Didn't know `-abc9` didn't get into 0.9.

Reply:
Yeah, I considered pinning to 0.9 but the nixpkgs Yosys package has always been on a git revision and there's already/still a bunch of stuff not in 0.9.